### ISSUE TITLE
How to move tracked TouchPoints from one TransformGesture to another?

### ISSUE THREAD

**Issue by Henning777** [fljot/Gestouch#40]
Hello,

I have a scenario where this is needed. I have 2 different TransformGestures with different targets and if TransformGesture A realized that it has more than 2 touch points it should be stopped and the touch points should be taken over by TransformGesture B. TG B should be able to perform scaling with these touch points there after. How can I achieve this?

I try to do these because of some performance issues on Starling. I realized, if I use 2 finger gestures on Starling on a Sprite object with some sub-objects (some more sprites as children) the performance deterioriates totally.

However if I use a transformGesture on a Sprite/Image object with no children the two finger gestures perform super great and everything runs very smooth.

If I try to scale from within TG A the target of TG B for some reasons (maybe  because of the target of TG A) it does not work out.

Thanks,
Henning

**Comment by fljot** [comment]
Nope, there's no such functionality. Yet it sort of was there long time ago, so I thought about this problem.
But still I didn't really understand your issue. Could you try to explain in in more details or maybe give some small project that demonstrates the issue? I think it might be solved some other way, without pushing touches from one gesture into another.
